snooty/rstspec.toml

```diff
@@ -712,10 +712,12 @@ content_type = "block"
 help = "Embed a quiz with a question argument, using the quizchoice directive to construct answer options."
 example = """.. quiz:: 
   :quiz-id: ${1:string}
-   
+  :quiz-date: ${1:string}
```

Comment:
the number here should actually probably be 3 not 1, but that's a very minor quibble.